c7n/resources/guardduty.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2015-2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from c7n.filters import ValueFilter, Filter
+from c7n.manager import resources
+from c7n.query import QueryResourceManager
+from c7n.utils import local_session, type_schema
+from c7n.actions import BaseAction
+
+
+@resources.register('guardduty-invitations')
+class GuardDutyInvitations(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'guardduty'
+        enum_spec = ('list_invitations', 'Invitations', None)
+        id = 'InvitationId'
+        name = 'Name'
+        dimension = None
+
+
+@resources.register('guardduty-detectors')
+class GuardDutyDetectors(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'guardduty'
+        enum_spec = ('list_detectors', None, None)
+        id = 'DetectorIds'
+        name = 'DetectorIds'
+        dimension = None
+
+
+@GuardDutyInvitations.filter_registry.register('list-invitations')
```

Comment:
list-invitations shouldn't be necessary as a filter as the resource itself can be filtered with a normal value filter:

```yaml
policies:
  - name: gd-invites
    resource: guardduty-invitations
    filters:
      - "AcountId": "000000000000"
```

Reply:
Got it! Sorry, I wasn't clear on default filter and thought custom filter is needed. I'll remove it.